android/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,2 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.reactnativestripesdk">
```

Comment:
For better backward compatibility, I recommend keeping package name here but lowering the gradle version to 7 `if (agpVersion.tokenize('.')[0].toInteger() >= 7) {`

This is what has been done for the most part on RN lib